logger/stos_logger.py

```diff
@@ -0,0 +1,21 @@
+import logging
+from logging.handlers import RotatingFileHandler
+import os
+
+LOGS_DIR: str = os.environ.get("LOGS_PATH", "/home/stos/")
+LOGGING_LEVEL: int = logging.INFO if os.environ.get("ENVIRONMENT") == "prod" else logging.DEBUG
+BACKUP_LOG_FILES: int = int(os.environ.get("BACKUP_LOG_FILES", 5))
+LOG_FILE_SIZE: int = int(os.environ.get("LOG_FILE_SIZE", 10 * 1024 * 1024))
+
+
+def get_logger(module_name: str) -> logging.Logger:
+    log_file = LOGS_DIR + module_name + '.log'
+
+    handler = RotatingFileHandler(log_file, maxBytes=LOG_FILE_SIZE, backupCount=BACKUP_LOG_FILES)
+    handler.setLevel(LOGGING_LEVEL)
+
+    logger = logging.getLogger(module_name)
+    logger.setLevel(LOGGING_LEVEL)
+    logger.addHandler(handler)
+
+    return logger
```

Comment:
I'm wondering if, since we already have the whole app written as some OOP amalgamation, shouldn't we wrap it into some class with common interface, and then inject it into our classes as a dependency. This way, it will stay consistent with the rest of our stuff (i.e. cache driver and such), and we'll have easier time configuring it.